paddle/math/MathFunctions.cpp

```diff
@@ -180,7 +180,6 @@ int getri<double>(const CBLAS_ORDER order,
                   const int lda,
                   const int* ipiv) {
   return dynload::PADDLE_DGETRI(order, N, A, lda, ipiv);
-  return 0;
```

Comment:
Yeah. I think we should, because usually this change requires a different group of reviewers, who are supposed to be familiar with math code, other than Android building.

But if you are sure this change is correct, I don't insist that we must separate it from this PR into a new PR.  You are welcome to make the decision. :-)